[2024-07-15T04:42:16.443+0000] {local_task_job_runner.py:120} INFO - ::group::Pre task execution logs
[2024-07-15T04:42:16.482+0000] {taskinstance.py:2076} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: etl_pipeline.clean_data manual__2024-07-15T04:42:10.842954+00:00 [queued]>
[2024-07-15T04:42:16.494+0000] {taskinstance.py:2076} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: etl_pipeline.clean_data manual__2024-07-15T04:42:10.842954+00:00 [queued]>
[2024-07-15T04:42:16.496+0000] {taskinstance.py:2306} INFO - Starting attempt 1 of 2
[2024-07-15T04:42:16.522+0000] {taskinstance.py:2330} INFO - Executing <Task(PythonOperator): clean_data> on 2024-07-15 04:42:10.842954+00:00
[2024-07-15T04:42:16.535+0000] {warnings.py:112} WARNING - /home/***/.local/lib/python3.12/site-packages/***/task/task_runner/standard_task_runner.py:61: DeprecationWarning: This process (pid=290) is multi-threaded, use of fork() may lead to deadlocks in the child.
  pid = os.fork()

[2024-07-15T04:42:16.536+0000] {standard_task_runner.py:63} INFO - Started process 292 to run task
[2024-07-15T04:42:16.541+0000] {standard_task_runner.py:90} INFO - Running: ['***', 'tasks', 'run', 'etl_pipeline', 'clean_data', 'manual__2024-07-15T04:42:10.842954+00:00', '--job-id', '229', '--raw', '--subdir', 'DAGS_FOLDER/demo.py', '--cfg-path', '/tmp/tmpc_v8f9mj']
[2024-07-15T04:42:16.542+0000] {standard_task_runner.py:91} INFO - Job 229: Subtask clean_data
[2024-07-15T04:42:16.609+0000] {task_command.py:426} INFO - Running <TaskInstance: etl_pipeline.clean_data manual__2024-07-15T04:42:10.842954+00:00 [running]> on host 37d6808b0cee
[2024-07-15T04:42:16.708+0000] {taskinstance.py:2648} INFO - Exporting env vars: AIRFLOW_CTX_DAG_EMAIL='your_email@example.com' AIRFLOW_CTX_DAG_OWNER='***' AIRFLOW_CTX_DAG_ID='etl_pipeline' AIRFLOW_CTX_TASK_ID='clean_data' AIRFLOW_CTX_EXECUTION_DATE='2024-07-15T04:42:10.842954+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='manual__2024-07-15T04:42:10.842954+00:00'
[2024-07-15T04:42:16.709+0000] {taskinstance.py:430} INFO - ::endgroup::
[2024-07-15T04:44:18.259+0000] {demo.py:53} INFO - Data Cleaned and Preprocessed
[2024-07-15T04:44:18.264+0000] {python.py:237} INFO - Done. Returned value was: None
[2024-07-15T04:44:18.264+0000] {taskinstance.py:441} INFO - ::group::Post task execution logs
[2024-07-15T04:44:18.281+0000] {taskinstance.py:1206} INFO - Marking task as SUCCESS. dag_id=etl_pipeline, task_id=clean_data, run_id=manual__2024-07-15T04:42:10.842954+00:00, execution_date=20240715T044210, start_date=20240715T044216, end_date=20240715T044418
[2024-07-15T04:44:18.360+0000] {local_task_job_runner.py:240} INFO - Task exited with return code 0
[2024-07-15T04:44:18.396+0000] {taskinstance.py:3503} INFO - 1 downstream tasks scheduled from follow-on schedule check
[2024-07-15T04:44:18.405+0000] {local_task_job_runner.py:222} INFO - ::endgroup::
